09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin:0;
+}
+main {
+    text-align:center;
+    background-image:url(../img/1.jpg);
+    width:100vw;
+    height:65vh;
+    color:white;
+    padding-top: 35vh;
+    font-family: sans-serif;
+    background-size: cover;
+    font-weight: 100;
+}
+
+.buttonservices {
+  padding: 10px 25px;
+  width: 90px;
+  margin: 0 auto;
+  font-size: .85rem;
+  background-color:#96ba55
+}
+
+.buttonservices * {
+  text-decoration:none;
+  color:white;
+}
+
+h1 {
+  font size: 1.75rem;
+}
+
+/*SERVICES STARTS HERE*/
+
+#main2 {
+  color:white;
+  font-family: sans-serif;
+  width:100vw;
+  height:100vh;
+  margin: 0;
+}
+
+#leftpanel {
+  height: 100vh;
+  width: 30vw;
+  float:left;
+  overflow: hidden;
+}
+
+.leftboxes {
+  height: 50vh;
+  width: 30vw;
+  float: left;
+  background-size: cover;
+  /*margin-left:1%;*/
+}
+
+h4 {
+  color: #96ba55;
+  font-weight: 400;
+}
+
+p {
+  font-family: serif;
+  line-height: 40px;
+}
+
+#growing {
+  background-image:url(../img/featured-image-3.jpg);
+  background-repeat: no-repeat;
+  width: 100%;
+  margin-bottom: 1.5%;
+  box-sizing: border-box;
+  text-align: left;
+  padding: 80px 0 0 50px;
```

Comment:
lols to photoshopping! easiest way if you can manage. it will be useful to know how to apply an opacity however, for pictures where you want a section of it with a dark background. make sure you still know how to do this.